installer/pyinstaller/build-linux.sh

```diff
@@ -47,15 +48,32 @@ echo "Installing PyInstaller"
 
 echo "Building Binary"
 cd src
+if ! [ "$nightly_build" = "" ]; then
+    echo "Updating samcli.spec with nightly build"
+    sed -i '' -e "s/\'sam\'/\'sam\-nightly\'/" installer/pyinstaller/samcli.spec
+fi
+echo "samcli.spec content is:"
+cat installer/pyinstaller/samcli.spec
 ../venv/bin/python -m PyInstaller -D --clean installer/pyinstaller/samcli.spec
 
 
 mkdir pyinstaller-output
-
-mv dist/sam pyinstaller-output/dist
+dist_folder="sam"
+if ! [ "$nightly_build" = "" ]; then
+    echo "using dist_folder with nightly build"
+    dist_folder="sam-nightly"
+fi
+echo "dist_folder=$dist_folder"
+mv dist/$dist_folder pyinstaller-output/dist
```

Comment:
Please `shellcheck` the script. This will break if `dist_folder` includes a space.

Reply:
Done. And checked with `shellcheck`.